modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -122,6 +123,10 @@ public class ApiClient {
     private OkHttpClient httpClient;
     private JSON json;
 
+    /**
+     * Visible so that unit tests don't have to mock java.util.logging.
+     */
+    HttpLoggingInterceptor.Logger debugLogger = HttpLoggingInterceptor.Logger.DEFAULT;
```

Comment:
if it isn't common to add tests to the same package, as me and I'll make a setter for this. We could then decide if to use literally the `HttpLoggingInterceptor.Logger` as the setter value, or a  `Consumer<string>`